Makefile

```diff
@@ -97,18 +99,9 @@ clean:
 	rm -rf $(BINDATA) $(OUT_DIR)
 
 local-image:
-ifdef USE_BUILDAH
-	buildah bud $(BUILDAH_OPTS) -t $(IMAGE_TAG) -f $(DOCKERFILE) .
-else
-	sudo docker build -t $(IMAGE_TAG) -f $(DOCKERFILE) .
-endif
+	podman build --no-cache -t $(IMAGE) -f $(DOCKERFILE) .
```

Comment:
Why the move to podman and not just buildah?

Reply:
Also I don't usually use rootless builds, so if it were up to me I'd keep/add sudo as well.

Reply:
> Is there a reason we want to remove a support for docker?

OpenShift 4.x is PodMan based, and there is no RHEL support for docker in the upstream releases,so we should move to the tools OCP is based on.

Reply:
@sjug supo option added

Reply:
I'm always doing rootless builds and I actually prefer them. :)

Reply:
update: rootless with buildah and root with podman

Reply:
What would be the reason to use `buildah` and `podman`?  I see the value in supporting `docker` at least for the time being, but I fail to see the reason to use both of `buildah` and `podman`.

Reply:
The sudo/sudo-less problem could be solved by something like: https://github.com/openshift/cloud-credential-operator/blob/1ea34db53e72fefeae29fa1da915644d8281caf1/Makefile#L76 

By default, I'd prefer sudo-less variant though unless there are objections.

Reply:
I like that approach, if others agree I can update the PR

Reply:
Yeah good call @jmencak, I like that sudo switch.

Reply:
```bash
DISTRO ?= $(shell if which lsb_release &> /dev/null; then lsb_release -si; else echo "Unknown"; fi)
# Default fedora to not using sudo since it's not needed
ifeq ($(DISTRO),Fedora)
	SUDO_CMD =
else # Other distros like RHEL 7 and CentOS 7 currently need sudo.
	SUDO_CMD = sudo
endif
```

the script for telling the os in that link doesn't work for me. looks like `lsb_release` doesn't come by default in fedora32.

thoughts?

Reply:
I think the above is completely unnecessary.  Users can add `SUDO_CMD` from the command line as an "external" `Makefile` variable.  I'd personally keep it empty by default and not add any more unnecessary code (like the one above) to the `Makefile`.  In our CI, the build works fine without sudo.  Never tested with sudo and I'd be reluctant to change this unless there's a strong reason for it.

Reply:
say no more, updating :)